veggieseasons/lib/main.dart

```diff
@@ -48,6 +56,9 @@ void setupWindow() {
   }
 }
 
+final _rootNavigatorKey = GlobalKey<NavigatorState>();
+final _shellNavigatorKey = GlobalKey<NavigatorState>();
```

Comment:
Different navigator key helps to open the detail pages inside or outside the shell navigator as a top modal screen